plugins/woocommerce/tests/e2e-pw/tests/analytics/analytics-data.spec.js

```diff
@@ -164,6 +164,80 @@ test.beforeAll( async ( { browser, restApi } ) => {
 			orderIds = response.data.create.map( ( order ) => order.id );
 		} );
 
+	// Reset Analytics Settings to their default values.
+	// Reset 'Excluded statuses' to default values.
+	await restApi
+		.post(
+			'wc-analytics/settings/wc_admin/woocommerce_excluded_report_order_statuses',
```

Comment:
Aside from this test, the other one that uses the `wc-analytics` endpoint is `customer-list.spec.js` but only as a wait criteria and not as an endpoint in a `restApi` request. So I didn't add a `WC_ANALYTICS` constant in `utils/api-client.js` but let me know if you think I should, and I'll add it on the next commit.